rust/capture/src/v0_endpoint.rs

```diff
@@ -351,6 +352,13 @@ pub async fn process_replay_events<'a>(
         .properties
         .remove("$session_id")
         .ok_or(CaptureError::MissingSessionId)?;
+
+    // Validate session_id is a valid UUID
+    let session_id_str = session_id.as_str().ok_or(CaptureError::InvalidSessionId)?;
+    if Uuid::parse_str(session_id_str).is_err() {
+        return Err(CaptureError::InvalidSessionId);
```

Comment:
Do these automatically become ingestion warnings? 

(Not a blocker!)

Reply:
They will be reported in `capture_events_dropped_total` with several other types of capture errors.

Reply:
👍 cool, not customer facing i don't think, but given we've run blobby for 2 years without needing this not the most important thing to report :)

Reply:
> A heads up to @pauldambra that to be clear, this means we'll drop this data and return a 400 error. Probably the SDK shouldn't generate these kind of Id's at all, and we should figure out why it does. Otherwise, +1.

@oliverb123 

the SDK generates valid UUIDv7 (or it bloody should!)

i think the issue in the incident is someone running burpsuite or something similar and seeing what happens if...

Reply:
but means I'm happy to 400 if the session replay UUID isn't UUIDlike

and it wouldn't work with the sessions table at all which enforces a valid UUIDv7 anyhoo